UnitTestProject1/Problem1Tests.cs

```diff
@@ -0,0 +1,196 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Problems.UnitTests
+{
+    [TestClass]
+    public class Probem1Tests
+    {
+        Problem1 problem;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            problem = new Problem1();
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_EmptyArray_Zero()
+        {
+            int[] arr = { };
+
+            long resultingSum = problem.SumOfEvenNumbers(arr);
+
+            Assert.AreEqual(resultingSum, 0);
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_OnlyZero_SumIsZero()
+        {
+            int[] arr = { 0 };
+
+            long resultingSum = problem.SumOfEvenNumbers(arr);
+
+            Assert.AreEqual(resultingSum, 0);
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_AllPositiveNoEvenNumbers_SumIsZero()
+        {
+            int[] arr = { 1, 3, 52241, 1201, 11 };
+
+            long resultingSum = problem.SumOfEvenNumbers(arr);
+
+            Assert.AreEqual(resultingSum, 0);
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_AllPositiveNoEvenNumbersDuplicates_SumIsZero()
```

Comment:
This test has no value, there is no point in testing duplicated even numbers.

Reply:
I had another look at the tests and removed those that checked for duplicates when there was no need to. Left one that encompasses the entire _duplicates_ scenarios.

`public void SumOfEvenNumbers_PositiveAndNegativeOddAndEvenDuplicates_SumIsMinus80()`